dev-lang/rust/rust-1.58.1.ebuild

```diff
@@ -377,6 +382,7 @@ src_configure() {
 		dist-src = false
 		remap-debuginfo = true
 		lld = $(usex system-llvm false $(toml_usex wasm))
+		llvm-libunwind = "$(usex elibc_musl system no)"
```

Comment:
does not seem to be correct flag to toggle?
I think it should be `llvm-libunwind = "$(usex llvm-libunwind system no)"

Reply:
or maybe even
`llvm-libunwind = "$(usex llvm-libunwind $(usex system-llvm system in-tree) no)" `

and not pul it as dep if system-llvm is not enabled

Reply:
```
$ grep llvm-libunwind profiles/use.desc 
llvm-libunwind - Use sys-libs/llvm-libunwind instead of sys-libs/libunwind
```
Please do not try to use this USE flag with different meaning.

If you want to have a USE flag for switching between using any libunwind library (`sys-libs/llvm-libunwind`, `sys-libs/libunwind`) or none, then `libunwind` USE flag can be used (it exists as local USE flag for few packages, e.g. `sys-libs/libcxxabi`, `dev-cpp/glog`).

If a USE flag would switch between system libunwind library or bundled libunwind library, then it could be called `system-libunwind`.

Reply:
system-llvm logic already covers it. don't see a problem with it or need to introduce yet another flag.
please stop offering unsolicited advice, not helping.

Reply:
The `llvm-libunwind = ` option in the config doesn't mean the same thing as the `llvm-libunwind` gentoo use flag. The `no` option on non-musl systems preserves the default for glibc systems (I really don't want to change what they use) and the `system` option dynamically links to either `llvm-libunwind` or `libunwind` based on whichever is installed in the system (which is what the use flag controls). The in-tree version was never used on gentoo before and it is still never used with this change. That would be a completely different PR/change.